### ISSUE TITLE
Function key not recognised

### ISSUE THREAD

**Issue by andydotxyz** [glfw/glfw#1785]
The macOS keyboard generates a code for the "fn" key, though GLFW has no support for function keys.
Is it possible to understand this key instead of returning an error?

This is a poor workaround that I have played with at the top of _glfwPlatformGetScancodeName :
```c
    if (scancode == 0x3F) { // macOS "fn" key
        return NULL; // No GLFW support for Function key
    }
```

Happy to explore better fixes for this.

**Comment by andydotxyz** [comment]
I also looked at adding the key code, but didn't know what to set it to:

```c
//    _glfw.ns.keycodes[0x3F] = GLFW_KEY_FUNCTION;
```

**Comment by elmindreda** [comment]
There is currently no key token to assign to the <kbd>Fn</kbd> key, but the error should finally be fixed with 86bf5698ecfdbe783135ffaf72972a8c562c0fdf. Thank you for the bug report!

**Comment by andydotxyz** [comment]
Thanks :)
